src/style.css

```diff
@@ -0,0 +1,231 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+}
+
+.title {
+  display: none;
+}
+
+.card-container {
+  /* padding-top: 51px; */
+  /* gap: 50px; */
+  /* margin: 0 auto; */
+  max-width: 1200px;
+  display: flex;
+  flex-wrap: wrap;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+}
+
+/* #region menu */
+.home-image-link {
+  height: 40px;
+  width: 40px;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1200px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.nav-item {
+  margin-right: 20px;
+  list-style: none;
+  position: relative;
+  text-transform: uppercase;
+  text-decoration: none;
+  align-items: flex-end;
+}
+
+.nav-item:last-child {
+  margin-right: 0;
+}
+
+.nav_tag {
+  display: inline-block;
+  margin: 0;
+  text-decoration: none;
+  font-size: 12px;
+  padding: 22.5px 0;
+}
+
+.nav_tag:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active:after {
+  /* margin: 0; */
+  position: absolute;
+  content: "";
+  height: 4px;
+  bottom: -1px;
+  background: #00acdc;
+  left: 0;
+  right: 0;
+  width: 100%;
+  border-radius: 8px;
+}
+
+/* #endregion menu */
+
+/* #region phonecard */
+.card {
+  display: flex;
+  flex: 1 0 35%;
+  margin: 0 25px 48px;
+  max-width: 200px;
+  flex-direction: column;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  justify-content: center;
+  align-items: center;
+}
+
+.mac-image {
+  padding: 32px 0 0;
```

Comment:
padding is not for this purpose, use margins instead (fix it here and all over your styles here)